packages/core/xhr/index.ts

```
@@ -497,15 +497,15 @@ export class FileReader {
 	public onprogress: (...args: any[]) => void;
 
 	private _readyState: number;
-	private _result: string | ArrayBuffer | null;
+	private _result: string | ArrayBuffer | SharedArrayBuffer | null;
 
 	private _listeners: Map<string, Array<Function>> = new Map<string, Array<Function>>();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)